scripts/datasheet-rendering/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "datasheet-rendering",
-  "version": "1.0.0",
```

Comment:
This doesn't refer to the datasheet-renderer package and hence has no effect here.

Reply:
True
What is that referring to?

Reply:
The script that integrates the renderer is also a node module. One that's not published in the registry but still it's a module.